plugins/techdocs-backend/src/service/router.ts

```diff
@@ -106,11 +106,16 @@ export async function createRouter({
   });
 
   router.get('/docs/:namespace/:kind/:name/*', async (req, res) => {
-    const { kind, namespace, name } = req.params;
     const storageUrl =
       config.getOptionalString('techdocs.storageUrl') ??
       `${await discovery.getExternalBaseUrl('techdocs')}/static/docs`;
 
+    res.redirect(`${storageUrl}${req.path.replace('/docs', '')}`);
+  });
+
+  //
```

Comment:
I see you wanted to write something for this endpoint, let's do it 😁

Reply:
Added a comment